validation/policies/io/konveyor/forklift/openstack/volume_based_test.rego

```diff
@@ -1,25 +1,25 @@
 package io.konveyor.forklift.openstack
 
-test_with_valid_image_format {
+test_image_based_vm {
 	mock_vm := {
 		"name": "test",
+		"imageID": "b749c132-bb97-4145-b86e-a1751cf75e21",
 		"image": {
 			"id": "b749c132-bb97-4145-b86e-a1751cf75e21",
-			"diskFormat": "qcow2",
 		},
 	}
 	results := concerns with input as mock_vm
-	count(results) == 0
+	count(results) == 1
 }
```

Comment:
aren't we supposed to have two concerns here when the status is not ACTIVE?

Reply:
which status? the test passes

Reply:
the status of the VM, we previously had:
```
mock_vm := {
		"name": "test",
		"status": "ACTIVE",
	}
```

Reply:
ah, that's because there's no status, if it's there and incorrect it would fail

Reply:
okok, in that case it's fine